server/migrations/versions/07859b6b370b_json_sample_sizes.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable=invalid-name
+"""JSON sample sizes
+
+Revision ID: 07859b6b370b
+Revises: 342f43b1c9d9
+Create Date: 2021-02-18 23:06:28.697346+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "07859b6b370b"
+down_revision = "342f43b1c9d9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "round_contest",
+        "sample_size",
+        type_=sa.JSON,
+        postgresql_using="json_build_object('key', 'custom', 'size', sample_size::int, 'prob', null)",
```

Comment:
I'm not familiar with this syntax, you've checked this will do the right thing for old sample sizes that were ints? I think it will, just making sure you're sure :)

Reply:
yeah i tried it out. i was actually surprised that it worked on the first try!